### PR TITLE
chore(console): translate i18n phrases

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Translate i18n phrases in console.

## Testing

Locally checked, but there might be mis-translations since it's generated by OpenAI

## Checklist

~~- [ ] `.changeset`~~
~~- [ ] unit tests~~
~~- [ ] integration tests~~
~~- [ ] necessary TSDoc comments~~

### CODE REVIEW

File: packages/phrases/src/locales/ja/errors/user.ts [hunk 1]
```diff
@@ -33,9 +33,7 @@ const user = {
   missing_mfa: 'MFAを追加してからサインインしてください。',
   totp_already_in_use: 'TOTPはすでに使用されています。',
   backup_code_already_in_use: 'バックアップコードはすでに使用されています。',
-  /** UNTRANSLATED */
   password_algorithm_required: 'Password algorithm is required.',
-  /** UNTRANSLATED */
   password_and_digest: 'You cannot set both plain text password and password digest.',
```
Comment (gao-sun) [inline packages/phrases/src/locales/ja/errors/user.ts:37]: these are still untranslated?

File: packages/phrases/src/locales/ja/translation/admin-console/api-resources.ts [hunk 2]
```diff
@@ -13,7 +13,6 @@ const api_resources = {
   default_api_label:
     'テナントごとにデフォルトのAPIを0または1つだけ設定できます。デフォルトのAPIが指定されている場合、認証リクエストでリソースパラメータを省略できます。その後のトークン交換は、デフォルトのAPIを対象として行われます。それにより、JWTが発行されます。<a>詳細を見る</a>',
   api_resource_created: 'APIリソース{{name}}が正常に作成されました',
-  /** UNTRANSLATED */
   invalid_resource_indicator_format: 'API indicator must be a valid absolute URI.',
```
Comment (gao-sun) [inline packages/phrases/src/locales/ja/translation/admin-console/api-resources.ts:16]: ditto. should have more like this

Comment (gao-sun) [reply packages/phrases/src/locales/ja/translation/admin-console/api-resources.ts:16]: it's not practical to check files one by one though. i'm good with some missing phrases

Comment (charIeszhao) [reply packages/phrases/src/locales/ja/translation/admin-console/api-resources.ts:16]: Discussed offline and we are OK to leave it as-is

**Review by gao-sun — APPROVED** [review]

**Pull request merged**